### PR TITLE
[wrap] Fixing jpeg

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -60,6 +52,31 @@ sys_bsd = bsd.contains(host_machine.system())
 sys_windows = windows.contains(host_machine.system())
 sys_osx = osx.contains(host_machine.system())
 
+if sys_windows
+  nuget_download = get_option('nuget-download')
+  if not nuget_download
+    message('needs -Dnuget-download=true to download deps')
+  endif
+  if cc.has_header_symbol('stdio.h', '_MSC_VER') and nuget_download
```
Comment (Coquinho) [inline meson.build:60]: This could be:
```suggestion
  elif cc.has_header_symbol('stdio.h', '_MSC_VER') 
```

File: meson.build [hunk 2]
```diff
@@ -60,6 +52,31 @@ sys_bsd = bsd.contains(host_machine.system())
...
+    endif
+
+    message(output)
+
+    subdir('win32-deps')
+  else
+    message('no nuget download because download failed or something')
+  endif
+else
+  message('not on windows')
```
Comment (Coquinho) [inline meson.build:77]: Do we need this message?

File: meson.build [hunk 3]
```diff
@@ -60,6 +52,31 @@ sys_bsd = bsd.contains(host_machine.system())
...
+    if r.returncode() != 0
+      errortxt = r.stderr().strip()
+      error(errortxt)
+    endif
+
+    message(output)
+
+    subdir('win32-deps')
+  else
+    message('no nuget download because download failed or something')
```
Comment (Coquinho) [inline meson.build:74]: Could this be a warning?

File: meson.build (hunk 1 above)
Comment (joaoantoniocardoso) [reply meson.build:60]: I think not, nuget should only be downloaded (line below) if `nuget_download` is set to true.

File: meson.build (hunk 2 above)
Comment (joaoantoniocardoso) [reply meson.build:77]: not at all.

File: meson.build (hunk 1 above)
Comment (Coquinho) [reply meson.build]: Yeah, how it is now it doesn't make sense.

File: meson.build [hunk 4]
```diff
@@ -60,6 +52,27 @@ sys_bsd = bsd.contains(host_machine.system())
...
+    endif
+
+    message(output)
+
+    subdir('win32-deps')
+  else
+    message('nuget-download disabled, if you want to use it to get dependencies you need to pass -Dnuget-download=true or set it in native-file-windows.txt')
+  endif
+endif
+
```
Comment (cauebs) [inline meson.build:75]: Is this still necessary if we're using a wrap?

Comment (joaoantoniocardoso) [reply meson.build:75]: At the point of this PR it was necessary to maintain Nuget working supposedly for the future. Turns out that we decided today to remove Nuget and then #334 does exactly that.

**Pull request merged**